Traceback (most recent call last):
  File "./cupy-test.py", line 12, in <module>
    c = cupy.identity(3,3).astype(cupy.float32)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/cupy/creation/basic.py", line 113, in identity
    return eye(n, dtype=dtype)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/cupy/creation/basic.py", line 93, in eye
    ret = zeros((N, M), dtype)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/cupy/creation/basic.py", line 179, in zeros
    a = cupy.ndarray(shape, dtype, order=order)
  File "cupy/core/core.pyx", line 118, in cupy.core.core.ndarray.__init__
  File "cupy/core/_dtype.pyx", line 62, in cupy.core._dtype.get_dtype_with_itemsize
TypeError: data type not understood
srun: error: gpu-0-0: task 0: Exited with exit code 1
